packages/terra-application-links/src/tabs/ApplicationTabs.scss

```diff
@@ -0,0 +1,169 @@
+:local {
+  .application-tabs {
+    display: flex;
+    flex: 1 1 auto;
+    flex-wrap: nowrap;
+    height: 100%;
+    justify-content: center;
+    padding-left: var(--terra-application-tabs-spacer-width, 2rem);
+    padding-right: var(--terra-application-tabs-spacer-width, 2rem);
+    position: relative;
+    width: 100%;
+  }
+
+  .tabs-container {
+    display: flex;
+    flex: 1 1 auto;
+    flex-wrap: nowrap;
+    height: 100%;
+    justify-content: center;
+    overflow: hidden;
+    position: relative;
+    width: calc(100% - 2 * var(--terra-application-tabs-spacer-width, 2rem));
+
+    &:focus {
+      outline: none;
+    }
+  }
+
+  .tabs-container.is-calculating {
+    visibility: hidden;
+  }
+
+  .tab span {
+    display: block;
+    left: 50%;
+    position: absolute;
+    top: 50%;
+    transform: translate3d(-50%, -50%, 0);
+    width: calc(100% - 2 * var(--terra-application-tabs-padding-horizontal, 1.071rem));
+  }
+
+  .tab,
+  .tab-menu {
+    span {
+      overflow: hidden;
+      text-overflow: ellipsis;
```

Comment:
This this desirable? `text-overflow: ellipsis;`

Reply:
Yes, we want to truncate all tabs, including the menu button tab. I could probably rename the class. As it might be confused for the styles of "tab-hidden".